src/manifest.json

```diff
@@ -3,7 +3,7 @@
     "name": "JIRA Template Injector",
     "author": "Redbrick",
     "short_name": "JTI",
-    "version": "1.1.1",
+    "version": "1.1.2",
```

Comment:
Looks like we've versioned this release 1.2.0 https://redbrickmedia.atlassian.net/projects/EXT/versions/19401/tab/release-report-all-issues (not 1.1.2)